examples/release_highlights/plot_release_highlights_1_1_0.py

```diff
@@ -161,3 +162,64 @@
 # - :class:`linear_model.GammaRegressor`
 # - :class:`linear_model.PoissonRegressor`
 # - :class:`linear_model.TweedieRegressor`
+
+# %%
+# MiniBatchNMF: an online version of NMF
+# --------------------------------------
+# The new class :class:`decomposition.MiniBatchNMF` implements a faster but less
+# accurate version of non-negative matrix factorization (:class:`decomposition.NMF`).
+# :class:`MiniBatchNMF` divides the data into mini-batches and optimizes the NMF model
+# in an online manner by cycling over the mini-batches, making it better suited for
+# large datasets. In particular, it implements `partial_fit`, which can be used for
+# online learning when the data is not readily available from the start, or when the
+# data does not fit into memory.
+import numpy as np
+from sklearn.decomposition import MiniBatchNMF
+
+rng = np.random.RandomState(0)
+n_samples, n_features, n_components = 10, 10, 5
+true_W = rng.uniform(size=(n_samples, n_components))
+true_H = rng.uniform(size=(n_components, n_features))
+X = true_W @ true_H
+
+nmf = MiniBatchNMF(n_components=n_components, random_state=0)
+
+for _ in range(10):
+    nmf.partial_fit(X)
+
+W = nmf.transform(X)
+H = nmf.components_
+X_reconstructed = W @ H
+
+print(
+    f"relative reconstruction error: ",
+    f"{np.sum((X - X_reconstructed) ** 2) / np.sum(X**2):.5f}",
+)
+
+# %%
+# BisectingKMeans: divide and cluster
+# -----------------------------------
+# The new class :class:`cluster.BisectingKMeans` is a variant of :class:`KMeans`, using
+# divisive hierarchical clustering. Instead of creating all centroids at once, centroids
+# are picked progressively based on a previous clustering: a cluster is split into two
+# new clusters repeatedly until the target number of clusters is reached, giving a
+# hierarchical structure to the clustering.
+from sklearn.datasets import make_blobs
+from sklearn.cluster import KMeans, BisectingKMeans
+import matplotlib.pyplot as plt
```

Comment:
```suggestion
```

We don't need this line because it was already imported.

Reply:
I assumed that we wanted each cell to be independant of the others such that a simple copy paste directly works

Reply:
Ah ok. It would be fine as well then.